specs/protocol/README.md

```diff
@@ -319,19 +276,29 @@ Currency amounts have type `DecimalString`, which is string containing a decimal
 ### `RFQ (Request For Quote)`
 > Alice -> PFI: "OK, that offering looks good. Give me a Quote against that Offering, and here is how much USD (payin currency) I want to trade for BTC (payout currency). Here are the credentials you're asking for, the payment method I intend to pay you USD with, and the payment method I expect you to pay me BTC in."
 
-| field          | data type                                         | required | description                                                                           |
-|----------------|---------------------------------------------------|----------|---------------------------------------------------------------------------------------|
-| `offeringId`   | string                                            | Y        | Offering which Alice would like to get a quote for                                    |
-| `payinAmount`  | [`DecimalString`](#decimalstring)                 | Y        | Amount of payin currency you want in exchange for payout currency                     |
-| `claims`       | string[]                                          | Y        | an array of claims that fulfill the requirements declared in an [Offering](#offering) |
-| `payinMethod`  | [`SelectedPaymentMethod`](#selectedpaymentmethod) | Y        | Specify which payment method to send payin currency.                                  |
-| `payoutMethod` | [`SelectedPaymentMethod`](#selectedpaymentmethod) | Y        | Specify which payment method to receive payout currency.                              |
+| field          | data type                                         | required | description                                                                                                                 |
+|----------------|---------------------------------------------------|----------|-----------------------------------------------------------------------------------------------------------------------------|
+| `offeringId`   | string                                            | Y        | Offering for which Alice would like to get a quote                                                                          |
+| `payinAmount`  | [`DecimalString`](#decimalstring)                 | Y        | Amount of payin currency you want in exchange for payout currency                                                           |
+| `claims`       | string[]                                          | N        | An array of claims that fulfill the requirements declared in an [Offering](#offering). Excluded from the signature payload. |
+| `claimsHash`   | string                                            | Y        | A hash [digest](#digest) of `claims`                                                                                        |
```

Comment:
Should `claimsHash` be not required?

Little bit confused what `claimsHash` would be if `claims` isn't present.

Reply:
Good catch -- typo on my part.